__tests__/integration.test.js

```diff
@@ -52,4 +52,17 @@ describe("/api/topics", () => {
         });
     });
 
+})
+
+describe('Being on api',()=>{
+    test("Responds with an object describing all the aviable endpoints in this API",()=>{
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({body})=>{
+            const {endpoint} = body
+            expect(typeof endpoint).toBe("object")
```

Comment:
this isn't helpful at all, it could be an empty object and still pass, but you want your tests to flag up any behaviour like that. Can you please add a test in to check the contents of the endpoints object, even if it's just requiring the file in the test suite and making sure that what comes back from the api looks the same